src/custom/Feedback/FeedbackButton.tsx

```diff
@@ -0,0 +1,250 @@
+          color: 'black'
+        }}
+      >
+        <Typography style={{ lineHeight: '2.5' }}>
+          Need help or have more feedback than fits here?
+          <br /> Meet with us.
+        </Typography>
+        <StyledLink
+          target="_blank"
+          href="https://calendar.google.com/calendar/appointments/schedules/AcZssZ3pmcApaDP4xd8hvG5fy8ylxuFxD3akIRc5vpWJ60q-HemQi80SFFAVftbiIsq9pgiA2o8yvU56"
+        >
+          Select a time convenient for you.
+        </StyledLink>
+      </div>
+    )
+  }
+];
+
+interface FeedbackComponentProps {
+  onSubmit: (data: { label: string; message: string }) => void;
+}
+
+const FeedbackComponent: React.FC<FeedbackComponentProps> = ({ onSubmit }) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [submitted, setSubmitted] = useState<boolean>(false);
+  const [category, setCategory] = useState<FeedbackDataItem | undefined>();
+  const [messageValue, setMessageValue] = useState<string | undefined>();
+  const feedbackTextRef = useRef<HTMLTextAreaElement>(null);
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+
+  const handleCheckboxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsChecked(event.target.checked);
+  };
+
+  const handleFeedback = () => {
+    setIsOpen(!isOpen);
+    setCategory(feedbackData[0]);
+    setSubmitted(false);
+    setMessageValue('');
+  };
+
+  const handleSubmit = () => {
+    setSubmitted(true);
+    if (messageValue && isChecked) {
+      onSubmit({ label: category?.label || '', message: messageValue });
+    }
+    setTimeout(() => {
+      setIsOpen(false);
+      setCategory(undefined);
+      setSubmitted(false);
+      setMessageValue('');
+    }, 2000);
+  };
+
+  return (
+    <Container isOpen={isOpen}>
+      {submitted ? (
+        <FeedbackMessage isOpen={isOpen}>
+          <SuccessIcon width={'32'} height={'32'} />
+          We got your concern. Thank you!
+        </FeedbackMessage>
+      ) : (
+        <>
+          <FeedbackButton onClick={handleFeedback}>Feedback</FeedbackButton>
+
+          <ModalCard
+            onClose={() => {}}
+            open={true}
+            closeComponent={
+              <CloseButton onClick={() => setIsOpen(false)}>
+                <CloseIcon fill="#F6F8F8" />
+              </CloseButton>
+            }
+            actions={
+              <div
+                style={{
+                  display: 'flex',
+                  alignItems: 'center'
+                }}
+              >
+                <div style={{ display: 'flex', alignItems: 'center' }}>
+                  <FormControlLabel
+                    label={''}
+                    style={{
+                      color: 'white',
+                      fontSize: '2px',
+                      marginLeft: '0px',
+                      marginRight: '5px'
+                    }}
+                    control={
+                      <Checkbox
+                        style={{ color: '#F6F8F8' }}
```

Comment:
@captain-Akshay  use directly from theme

Reply:
yes that's what i m working on.